.github/workflows/scalar-functional-tests.yml

```diff
@@ -1,9 +1,10 @@
 name: Scalar Functional Tests
 
 env:
-  SCALAR_REPOSITORY: dscho/scalar
-  SCALAR_REF: vfs-2.32.0
+  SCALAR_REPOSITORY: derrickstolee/scalar
+  SCALAR_REF: test-scalar-in-c
```

Comment:
This is good. We will update the change after we update `microsoft/scalar:main` with the necessary changes to make it work.